src/components/table/base-table/base-table.js

```diff
@@ -329,51 +339,81 @@ export default class BaseTable extends React.Component {
       </Cell>
     );
 
-  itemRenderer = renderParams => {
+  renderItemCell = renderParams => {
     // `columnDefinition` is the original column-definition from where you defined the table
     // `renderParams` holds similar looking-information but is the data provided by "react-virtualized"
     const columnDefinition = this.columns[renderParams.columnIndex];
+    if (renderParams.rowIndex === 0) {
+      return (
+        <ClassNames>
+          {({ css: makeClassName }) => (
```

Comment:
Given that we need to merge the class names provided to the component with some default ones, we need to use this workaround to generate a classname from emotion, and merge that with the user defined ones.